api/batch/resource.go

```diff
@@ -71,25 +72,22 @@ var (
 		OnFailureRetryInterval:           &failureRetryInterval,
 	}
 
-	defaultToleration = corev1.Toleration{
-		Key:      "batch-job",
-		Operator: corev1.TolerationOpEqual,
-		Value:    "true",
-		Effect:   corev1.TaintEffectNoSchedule,
-	}
-
 	defaultHadoopConf = map[string]string{
 		hadoopConfEnableServiceAccountKey: hadoopConfEnableServiceAccount,
 		haddopConfServiceAccountPathKey:   haddopConfServiceAccountPath,
 	}
-
-	defaultNodeSelector = map[string]string{
-		"node-workload-type": "batch",
-	}
```

Comment:
These effectively removes the default tolerations and node selectors. I'm not sure if we'd like to keep this, and if we do, maybe we can consider moving them to the CaraML chart?

Reply:
should be defaulted as part of chart